### DISCUSSION TITLE
I am not able to push my code.

### DISCUSSION THREAD

**Discussion by khushali42** [community/community#58419] [New to GitHub]
fatal: unable to access 'https://github.com/khushali42/personal-portfolio.git/': Failed to connect to 127.0.0.1 port 1089 after 2031 ms: Couldn't connect to server

getting this error for every git command
tried every possible solution but didn't worked

**Comment by nirav-gajera** [comment]
-Check your proxy settings: If you're behind a firewall or using a proxy, make sure your Git configuration is set up correctly to work with your network settings. You can try updating your Git configuration to use the appropriate proxy settings.

```
git config --global http.proxy http://proxy.example.com:8080
git config --global https.proxy https://proxy.example.com:8080
```

Replace proxy.example.com and 8080 with the actual proxy server and port.

-Try using SSH instead of HTTPS: If you have set up SSH keys with your GitHub account, you can try cloning the repository using the SSH URL instead of the HTTPS URL.

`git clone git@github.com:khushali42/personal-portfolio.git
`

**Reply by airtower-luna** [reply]
It definitely looks like a broken proxy setting, but:

* A proxy might also be configured (more commonly, even) via environment variables, check `http_proxy`, `https_proxy`, and `all_proxy`.
* Depending on if you *want* to use any proxy, instead of changing it might be better to unset them.
